frontend/lib/src/components/elements/Html/Html.tsx

```diff
@@ -16,7 +16,7 @@
 
 import React, { memo, ReactElement, useEffect, useRef, useState } from "react"
 
-import DOMPurify from "dompurify"
+import dompurify from "dompurify"
```

Comment:
We need to rename this to prevent a linting error since it now also exports a type with the name `DOMPurify`.